src/api/entities/HistoricalMultiSigProposal.ts

```diff
@@ -0,0 +1,177 @@
+import BigNumber from 'bignumber.js';
+
+import { Context, Entity } from '~/internal';
+import { multiSigProposalVotesQuery } from '~/middleware/queries';
+import { Query } from '~/middleware/types';
+import { MultiSigProposalVote, SignerType } from '~/types';
+import { Ensured } from '~/types/utils';
+import { middlewareEventDetailsToEventIdentifier, signerValueToSigner } from '~/utils/conversion';
+
+export type MiddlewareMultiSigProposal = {
+  id: string;
```

Comment:
```suggestion
```
This is not needed to be exposed to end user. ID from subquery is something we would use to just query stuff. It is generally a combination of a few values which would confuse the user to about what it is